tests/mbed/mbed.edl

```diff
@@ -2,14 +2,47 @@
 // Licensed under the MIT License.
 
 enclave {
-     struct mbed_args {
+    include "fcntl.h"
```

Comment:
Nice use of EDL capabilities.

Reply:
I thought that it was pretty neat that I could use system types in the EDL.